content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -4,185 +4,229 @@ Process receipts and deliveries in two steps
 
 Once the sales order is confirmed, a picking order is generated and ready to process.
 
-Navigate to the :menuselection:`Inventory app`, and locate the :guilabel:`Pick` task card on the
-:guilabel:`Inventory Overview` dashboard. This reveals a list of all pickings to process.
+To complete the picking, navigate to the :guilabel:`Inventory app`, and locate the :guilabel:`Pick`
+task card on the :guilabel:`Inventory Overview` dashboard.
+
+Click the :guilabel:`# To Process` button on the task card. This reveals a list of all pickings to
+process.
 
 Click on the picking (WH/PICK) operation associated with the sales order to reveal the picking
 order.
 
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-picking-form.png
+   :align: center
+   :alt: Picking order form for products included in sales order.
+
 Click :guilabel:`Check Availability` on the picking order form. If the product is in stock, Odoo
 automatically reserves the product. Quantities can also be manually set by changing the value in the
 :guilabel:`Quantity` column to match the value in the :guilabel:`Demand` column.
 
-Once ready, click :guilabel:`Validate` to complete the picking and move the product from
+Once ready, click :guilabel:`Validate` to complete the picking, and move the product from
 :guilabel:`WH/Stock` to :guilabel:`WH/Output.`
 
-.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-picking-form.png
-   :align: center
-   :alt: Picking operation form moving product to WH/Output.
-
-Process the delivery
---------------------
-
-Once the picking has been processed, the delivery order is generated, and can be processed.
+Process delivery
+----------------
 
-Navigate back to the original sales order by going to :menuselection:`Sales app`, and selecting the
-associated sales order.
+Once the picking is validated, a delivery order is created and ready to process.
 
-The :guilabel:`Delivery` smart button now indicates there are two transfers. Click the
-:guilabel:`Delivery` smart button to reveal each transfer tied to this sales order: the completed
-picking, and the new delivery (WH/OUT) operation. Click the delivery operation to open the delivery
-order.
+To view the delivery order, navigate back to the :guilabel:`Inventory Overview` page, via the
+breadcrumbs, and locate the :guilabel:`Delivery Orders` task card.
 
-Once ready, click :guilabel:`Validate` to complete the delivery.
+Click the :guilabel:`# To Process` button on the task card to reveal a list of all delivery orders
+to process, and select the order associated with the previously validated picking.
 
 .. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-order.png
    :align: center
-   :alt: Delivery operation form moving product to customer location.
+   :alt: Delivery order form for products ordered by customer.
+
+To deliver the products, change the value in the :guilabel:`Quantity` field to match the ordered
+quantity in the :guilabel:`Demand` field.
```

Comment:
```suggestion
To deliver the products, change the value in the :guilabel:`Quantity` field to match the ordered
quantity in the :guilabel:`Demand` field.
```

This was done automatically for me.

Reply:
In 17.2 it's not done automatically, and only after completion of the previous steps

Reply:
Tim and I are getting the same result here, as well